data/mods/Aftershock/items/afs__items.json

```diff
@@ -170,11 +170,26 @@
     "symbol": "u",
     "color": "light_green",
     "use_action": { "type": "deploy_furn", "furn_type": "f_atomic_butter_churn" },
-    "qualities": [ [ "CONTAIN", 1 ], [ "CHURN", 2 ] ],
     "to_hit": -2,
     "bashing": 5,
     "material": [ "plastic", "steel" ],
     "//": "I figured bionic preppers would have a sci-fi solution to some low tech problems.",
     "flags": [ "LEAK_DAM", "RADIOACTIVE", "DURABLE_MELEE" ]
+  },
+  {
+    "id": "atomic_butter_churn",
```

Comment:
it's very *very* easy to mix this id up with the actual (non-pseudo) item. The only difference is an underscore and who will know that "atomic_butter_churn" is the pseudo item and "atomic_butterchurn" is the actual item? I suggest you add some hint like "pseudo_" or similar to the id.

Reply:
I think I've resolved that. Let me know your thoughts.